### PR TITLE
Fix typos, rephrase words and improve readability

### PR DESCRIPTION
**Pull request by fschmtt**

### CODE REVIEW

**Review by shyim — APPROVED** [review]

File: guides/installation/flex.md [hunk 1]
```diff
@@ -10,38 +10,54 @@ To create a new Shopware project, run the following command:
 composer create-project shopware/production:dev-flex <project-name>
 ```
 
-This creates a new project in the `<project-name>` directory. The `dev-flex` version constraint installs the latest version of the Shopware. This constraint `dev-flex` is removed when the template is generally available.
+This creates a new project in the `<project-name>` directory. The `dev-flex` version constraint installs the latest version of Shopware. The constraint `dev-flex` will be removed when the template is generally available.
```
Comment (fschmtt) [inline guides/installation/flex.md:13]:
```suggestion
This creates a new project in the `<project-name>` directory. The `dev-flex` version constraint installs the latest version of Shopware. The constraint `dev-flex` will be removed once the template is generally available.
```

File: guides/installation/flex.md [hunk 2]
```diff
@@ -10,38 +10,47 @@ To create a new Shopware project, run the following command:
 composer create-project shopware/production:dev-flex <project-name>
 ```
 
-This creates a new project in the `<project-name>` directory. The `dev-flex` version constraint installs the latest version of the Shopware. This constraint `dev-flex` is removed when the template is generally available.
+This creates a new project in the `<project-name>` directory. The `dev-flex` version constraint installs the latest version of Shopware. The constraint `dev-flex` will be removed once the template is generally available.
 
-The template contains all Shopware bundles like `shopware/administration`, `shopware/storefront`, `shopware/elasticsearch`. If you don't need one of them, you can uninstall it with:
-`composer remove shopware/<bundle-name>`.
+The template contains all Shopware bundles like `shopware/administration`, `shopware/storefront` and `shopware/elasticsearch`. If you don't need any of them, you can uninstall it with:
```
Comment (sushmangupta) [inline guides/installation/flex.md:15]:
```suggestion
The template contains all Shopware bundles like `shopware/administration`, `shopware/storefront` and `shopware/elasticsearch`. If you don't need any, then you can uninstall them with:
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]
@fschmtt,
"composer" is always written in small case throughout the dev doc, unlike in this article. Do you agree with this?
Else, we'll need to talk about it.

**Comment by fschmtt** [comment]
> @fschmtt, "composer" is always written in small case throughout the dev doc, unlike in this article. Do you agree with this? Else, we'll need to talk about it.

"Composer", is, like Symfony, Shopware etc., a "brand" name and should thus be written capitalized in regular text passages IMO.

I think we didn't really pay attention to this (not only for Composer but other "brand" names, too) so far, but IMO we should.

WDYT @sushmangupta?

**Review by sushmangupta — APPROVED** [review]

**Comment by sushmangupta** [comment]
@fschmtt, Yes I totally agree with you. I will make this correction overall. 
Let me know if you observe any other brand names that need a correction in our dev docs.

**Pull request merged**